modules/op-resolver-git-override-scm.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-git-override-scm_{context}"]
+= Overriding the Git resolver for the authenticated SCM API
```

Comment:
This module does not seem to work. It is not about "overriding", the TektonConfig process is for "configuring" the SCM API for the cluster. You can then "override" it when using the resolver to specify a particular task or pipeline.

Reply:
Aha, I see what happened here, you start with TektonConfig and switch to a pipeline run spec later.

I suggest reworking this part entirely. TektonConfig is already documented, no need to say anything more about it. And the "overriding" part is about using, not configuring, the resolver.

I see two options:

- expand op-resolver-git-scm.adoc to cover the override use case
- have a separate module similar to op-resolver-git-scm.adoc but specifically for the override use case